sound/soc/intel/boards/sof_sdw.c

```diff
@@ -555,8 +555,8 @@ static struct sof_sdw_codec_info codec_info_list[] = {
 				.dai_name = "rt711-sdca-aif1",
 				.dai_type = SOF_SDW_DAI_TYPE_JACK,
 				.dailink = {SDW_JACK_OUT_DAI_ID, SDW_JACK_IN_DAI_ID},
-				.init = sof_sdw_rt_sdca_jack_init,
-				.exit = sof_sdw_rt_sdca_jack_exit,
+				.init = soc_sdw_rt_sdca_jack_init,
+				.exit = soc_sdw_rt_sdca_jack_exit,
```

Comment:
I would really try to be consistent and use the same 'asoc_sdw' prefix for all helpers and variables

Reply:
will follow the 'asoc_sdw' prefix for consistency